core/LSU.cpp

```diff
@@ -290,7 +296,7 @@ namespace olympia
     }
 
     void LSU::getAckFromCache_(const MemoryAccessInfoPtr &updated_memory_access_info_ptr){
-        cache_hit_ = updated_memory_access_info_ptr->getCacheState() == MemoryAccessInfo::CacheState::HIT;
+        cache_hit_ = updated_memory_access_info_ptr->isCacheHit();
```

Comment:
I think we can event get rid of `cache_hit_` and `mmu_hit_`

Reply:
I would need it again in the following Pr #85 to decide if I need to  stall the pipe or let it continue as usual

Reply:
ok